lib/requests/reset-storage.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+
+const debug = require('./../debug').authentication
+
+class ResetStorage {
+
+    /**
+     * Handles a clearCookie GET request on behalf of a middleware handler, clears
+     * the cookies from the application.
+     * Usage:
+     *
+     *   ```
+     *   app.get('/reset', ResetStorage.reset)
+     *   ```
+     *
+     * @param req {IncomingRequest}
+     * @param res {ServerResponse}
+     */
+    static reset(req, res) {
+        debug('Reset Cookies & Local Storage from the application')
+        const cookies = req.cookies;
+        for (let prop in cookies) {
+            if (!cookies.hasOwnProperty(prop)) {
+                continue;
+            }
+            res.cookie(prop, '', { expires: new Date(0) })
+        }
+        // if (typeof (localStorage) !== undefined) {
```

Comment:
localStorage.clear would have to be run on front-end code, not back-end code such as this.

In that regard, cookies could also be removed via the front-end, but this should work fine as well.